ltext.py

```diff
@@ -600,9 +592,9 @@ def equals(self, other):
         return (
                 self.text == other.text
                 and all(
-                    (isinstance(a, Label) and a == b)
```

Comment:
It seems like a pycharm auto reformat.

I doubt It is making indentation worse. I like the one before reformat.

I'll check PEP8 about indentation later.